app/controllers/hosts_controller.rb

```diff
@@ -712,15 +711,4 @@ def merge_search_filter(filter)
   def offer_to_overwrite_conflicts
     @host.overwrite = "true" if @host.errors.any? and @host.errors.are_all_conflicts?
   end
-
-  # :root_pass and bmc :password should not update the Host and Nic passwords when blank
-  # This method removes them from the params hash to avoid any processing of these attributes.
-  def cleanup_passwords
```

Comment:
not need anymore, the root_pass and the BMC password in interfaces_attributes are disabled in default